commands/anime/baka.js

```diff
@@ -0,0 +1,41 @@
+const { MessageEmbed } = require("discord.js");
+const config = require("../../configs/config.json");
+const nekos = require("nekos.life");
+const {
+  sfw: { baka },
+} = new nekos();
+
+module.exports = {
+  config: {
+    name: "baka",
+    description: "Get's a reaction for baka!",
+    aliases: ["BAKA", "Baka"],
+    usage: "<user>",
+    accessableby: "",
+  },
+  run: async (client, message, args) => {
+    const { url } = await baka().catch(() => {});
+
+    if (!url) return message.channel.send(`Could not connect to nekos.life`);
+
+    const embed = new MessageEmbed();
+    if (
+      message.mentions.members.size &&
+      message.mentions.members.first().id === client.user.id
+    ) {
+      return message.channel.send(`Chigau! Anata wa baka desu!`);
+    } else if (
+      message.mentions.members.size &&
+      message.mentions.members.first().id === message.author.id
+    ) {
+      return message.channel.send(`Seriously?`);
+    } else if (message.mentions.members.size) {
+      return message.channel.send(
+        embed
+          .setColor("GREY")
```

Comment:
Could you set the embed color to the one in config for this and the other files? It would be `config.embedcolor`

Reply:
how about I set it to random? should work for ya I guess??

Reply:
Oh well, If you check the other commands, It has the embed color set to one specific color. So It would be weird having these commands have a random color and the others as the one in config 😆

Reply:
Yeah, gotta. I actually am blind not seeing that clearly xD